src/gamekit.cs

```diff
@@ -853,6 +853,21 @@ interface GKLocalPlayer
 		Action<NSViewController, NSError> AuthenticateHandler { get; set; }
 #endif
 
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("isPresentingFriendRequestViewController")]
+		bool IsPresentingFriendRequestViewController { get; }
+
+#if !MONOMAC
```

Comment:
You can add something like this at the top of the file:

```csharp
using NSWindow = Foundation.NSObject;
```

here: https://github.com/xamarin/xamarin-macios/blob/a7c9099153bb34bee78a26f925fa0fc545c2a4c0/src/gamekit.cs#L26-L29

This way the API declaration will compile just fine without the conditional logic.

You shouldn't need something for `UIViewController`, since it's already there:

https://github.com/xamarin/xamarin-macios/blob/a7c9099153bb34bee78a26f925fa0fc545c2a4c0/src/gamekit.cs#L25

Reply:
Okay that makes sense why the UIViewController was not giving me errors! Thanks!